### ISSUE TITLE
output "modifiedFiles.txt" with the file size comparison

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/CompareDirectoriesGradlePlugin#65]
derived from https://github.com/aogan-office/aomori-gankaikai-HP/issues/53

Ond day I got a `differences.json` like this:

```
{
  "dirA" : "C:\\Users\\aogan\\github-aogan\\aomori-gankaikai-HP\\site\\source",
  "dirB" : "C:\\Users\\aogan\\github-aogan\\aomori-gankaikai-HP\\site\\wgot\\www.aomori-gankaikai.jp",
...
  "modifiedFiles" : [
    "about\\index.html",
    "aboutus\\greeting.html",
    "aboutus\\history.html",
    "aboutus\\index.html",
    "aboutus\\part.html",
    "contact\\index.html",
    "contact\\index1.html",
    "css\\about.css",
    "css\\common\\general.css",
    "css\\common\\layout.css",
    "css\\eyehealth.css",
    "css\\school1.css",
    "css\\site-style-large.css",
    "css\\site-style-small.css",
    "css\\site-style.css",
    "eyebank\\index.html",
    "health\\eyehealth.html",
    "health\\index.html",
    "institution\\aomori.html",
    "institution\\gosyogawara.html",
    "institution\\hachinohe.html",
    "institution\\hirosaki.html",
    "institution\\index.html",
    "institution\\mutsu.html",
    "institution\\towada.html",
    "js\\head.js",
    "js\\jquery-3.6.0.min.js",
    "js\\swicher.js",
    "js\\swindow.js",
    "link\\index.html",
    "member\\area\\aomori.html",
    "member\\area\\hachinohe.html",
    "member\\area\\hirosaki.html",
    "member\\area\\index.html",
    "member\\dyschromatopsia_doctor.html",
    "member\\index.html",
    "member\\magazine.html",
    "member\\movie.html",
    "member\\source.html",
    "school\\dyschromatopsia.html",
    "school\\index.html",
    "school\\school1.html",
    "school\\school2.html",
    "school\\school3.html"
  ]
}

```

I could see the list of file names that are present in both dirA and dirB, and have some difference.

But What sort of difference?

I want to know the size information of each lines in both dirA and dirB. For example:

```
...
  "modifiedFiles" : [
    ["about\\index.html", 1000, 1200],
...
```

But JSON syntax rule restricts the format.
Better to emit another file in an arbitrary text, like

`modifiedFiles.txt`
```
about\\index.html                                                                 1000 -->  1200
aboutus\\index.html                                                             2000 --> 2345
... 
```

**Comment by kazurayam** [comment]
changed the task to generate "nameStatusList.tsv"

```
name	status	timestampA	<>	timestampB	sizeA	<>	sizeB
d.txt	-	2024-03-29T09:45:03.67424	<	2024-03-29T09:45:03.676121	1	=	1
e.txt	-	2024-03-29T09:45:03.674585	<	2024-03-29T09:45:03.676499	1	=	1
j.txt	A	-	-	2024-03-29T09:45:03.676875	-	-	1
sub/f.txt	-	2024-03-29T09:45:03.674951	<	2024-03-29T09:45:03.677242	1	=	1
sub/g.txt	M	2024-03-29T09:45:03.675207	<	2024-03-29T09:45:03.677459	1	<	12
sub/h.txt	A	-	-	2024-03-29T09:45:03.677651	-	-	1
sub/i.txt	D	2024-03-29T09:45:03.675413	-	-	1	-	-
このファイルはシフトJISだよん.txt	M	2024-04-10T20:26:03.362659	<	2024-04-10T20:26:58.764097	32	<	46
取扱説明書.md	-	2024-04-08T22:54:19.303185	<	2024-04-08T22:54:19.303569	20	=	20

```

This is a Tab-Separated-Value text, which you can open in Excel or Number of Mac.

<img width="1029" alt="image" src="https://github.com/user-attachments/assets/9d893107-6777-4e3d-ad1e-4eb078ae190c">

**Comment by kazurayam** [comment]
Had better swap the timestamp and size.

name > status > sizeA > sizeB > timestampA > timestampB

**Comment by kazurayam** [comment]
published the plugin version 0.2.10
